spec/models/passenger_spec.rb

```diff
@@ -3,125 +3,161 @@
 require 'rails_helper'
 
 RSpec.describe Passenger do
-  context 'testing mailers upon creation' do
-    it 'correct email is sent upon creation of active passenger' do
-      mail = ActionMailer::MessageDelivery.new(PassengerMailer, :notify_active)
-      expect(PassengerMailer).to receive(:notify_active).and_return mail
-      expect(PassengerMailer).not_to receive(:notify_pending)
-      expect(PassengerMailer).not_to receive(:notify_archived)
-      expect(mail).to receive(:deliver_now).and_return true
+  let(:passenger) { create(:temporary_passenger, :with_note) }
+
+  before do
+    @passenger = create(:temporary_passenger, :with_note)
```

Comment:
is this `before` needed at all? definitely not the instance variable

Reply:
'Tis not. My strategy was to add the `let`, progressively change the usage in the specs, then remove the instance variable. Just forgot that last step.